subsys/bluetooth/controller/ll_sw/ull_conn.c

```diff
@@ -698,7 +698,13 @@ uint8_t ll_apto_get(uint16_t handle, uint16_t *apto)
 		return BT_HCI_ERR_UNKNOWN_CONN_ID;
 	}
 
-	*apto = conn->apto_reload * conn->lll.interval * 125U / 1000;
+	if (conn->lll.interval > 5U) {
+		*apto = conn->apto_reload * conn->lll.interval *
+			CONN_INT_UNIT_US / 10000U;
+	} else {
+		*apto = conn->apto_reload * (conn->lll.interval + 1U) *
+			CONN_LOW_LAT_INT_UNIT_US / 10000U;
+	}
```

Comment:
I think there are some magic numbers (`5U` and `10000U` that are either defined or should be defined. The division by `10000U` for a microsecond value would appears to be a translation to a different unit, and could be a macro or something.

Reply:
Host uses hardcoded value `6` and i could not easily find a define for minimum BLE Connection Interval!  The timeout are in 10 ms unit, same here, do we have those defines?

Reply:
Yeah, for some reason they were not defined before https://github.com/zephyrproject-rtos/zephyr/pull/79584 

I don't think we have any conversion macros between ms and units of 10ms, so we usually just do it like here, but with a comment stating that we are converting to a centiseconds (10ms). 

It could help to replace `10000` with `USEC_PER_MSEC * 10` as well